sw/device/lib/dif/dif_rv_core_ibex.h

```diff
@@ -76,6 +76,36 @@ typedef struct dif_rv_core_ibex_addr_translation_region {
   dif_rv_core_ibex_addr_translation_pair_t dbus;
 } dif_rv_core_ibex_addr_translation_region_t;
 
+/**
+ * Ibex error status detected by `rv_core_ibex` peripheral.
+ */
+typedef enum dif_rv_core_ibex_error_status {
+  /**
+   * Register transmission integrity error.
+   */
+  kDifRvCoreIbexErrorStatusRegisterTransmissionIntegrity = 1 << 0,
+
+  /**
+   * Response integrity error.
+   */
+  kDifRvCoreIbexErrorStatusFatalResponseIntegrity = 1 << 8,
```

Comment:
If planning to leave this way, can you add `static_assert` entries in the `.c` to check against the hw generated constant values?

Reply:
Done